### PR TITLE
Migrate multi function

### PR DESCRIPTION
**Pull request by qingxinhome**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [x] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

issue #2565

**What this PR does / why we need it:**

According to plan2 requirements, migrating multivariate function. the following multivariate function are migrated：
lpad,  rpad，round,  ceil

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

File: pkg/sql/plan2/function/builtin/multi/pi.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	"errors"
+	"github.com/matrixorigin/matrixone/pkg/container/nulls"
+	"github.com/matrixorigin/matrixone/pkg/container/types"
+	"github.com/matrixorigin/matrixone/pkg/container/vector"
+	"github.com/matrixorigin/matrixone/pkg/vectorize/pi"
+	"github.com/matrixorigin/matrixone/pkg/vm/process"
+)
+
+func Pi(lv []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
+	if len(lv) != 0 {
```
Comment (m-schen) [inline pkg/sql/plan2/function/builtin/multi/pi.go:27]: redundant check

**Review by m-schen — COMMENTED** [review]
lgtm, but some conflict with the pr #2700
you all implement the Pi() and etc_timestamp().
please do some check and maybe one of you should remove your implement.

some check is redundant. we will do check on type-check-function for arguments' type and number.

File: pkg/sql/plan2/function/builtin/multi/pi.go (hunk 1 above)
Comment (qingxinhome) [reply pkg/sql/plan2/function/builtin/multi/pi.go:27]: You're right. It's a redundancy check. Let me correct it

**Comment by qingxinhome** [comment]
> lgtm, but some conflict with the pr #2700 you all implement the Pi() and etc_timestamp(). please do some check and maybe one of you should remove his implement.
> 
> some check is redundant. we will do check on type-check-function for arguments' type and number. @qingxinhome

OK，I will negotiate this matter

**Pull request merged**